spec/rails_helper.rb

```diff
@@ -28,10 +28,10 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 
-Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
-Dir[Rails.root.join("spec/system/shared_examples/**/*.rb")].each { |f| require f }
-Dir[Rails.root.join("spec/models/shared_examples/**/*.rb")].each { |f| require f }
-Dir[Rails.root.join("spec/jobs/shared_examples/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
```

Comment:
why the sort?

Reply:
For this cop ==> https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_lint.md#lintnondeterministicrequireorder